### ISSUE TITLE
Getting several corrupt screenshots on hover of a panel tab

### ISSUE THREAD

**Issue by ohlogic** [mate-desktop/mate-panel#1157]
#### Expected behaviour
When I hover over the mate panel and a window tab I should get a screenshot of the window of the application without corruption of the screenshot.

#### Actual behaviour
When I hover over the mate panel and a window tab I almost always get a corrupt screenshot.  
example1
![Screenshot at 2020-11-05 15-35-03](https://user-images.githubusercontent.com/8847334/98293873-04647e80-1f7d-11eb-982c-944cf1f1871f.png)

example2
![Screenshot at 2020-11-05 15-37-25](https://user-images.githubusercontent.com/8847334/98293901-0e867d00-1f7d-11eb-81ef-5342155d1a2c.png)

#### Steps to reproduce the behaviour
I simply hover over the panel of a window tab and the corruption of a screenshot occurs.

#### MATE general version
1.24.0

#### Package version
mate-panel 1.24.0-2

#### Linux Distribution
Linux Mint 20

**Comment by lukefromdc** [comment]
I was unable to duplicate this either with marco or with compiz-reloaded as the window manager. I am however running Debian Unstable (recently updated) and mate 1.25 local builds from git master. This is with "show thumbnails on hover" selected in the window list applet prefs.

**Comment by raveit65** [comment]
Ok, closing.

**Comment by ohlogic** [comment]
Seems to be resolves in MATE 1.26

 
 
  On Sun, Jul 23, 2023 at 3:07 PM, ***@***.***> wrote:   


Ok, closing.

**Comment by fpeelo** [comment]
I'm getting this problem, and I'm wondering if the window manager is to blame.

On previous versions of Mint, it happened on one laptop but it wasn't a machine I cared much about - now it's happening on the 2 machines I use the most...

```
$ inxi -Sz
System:
  Kernel: 5.15.0-131-generic x86_64 bits: 64 Desktop: MATE 1.26.0
    Distro: Linux Mint 21.3 Virginia

```
It started when I changed the Window Manager. Why did I do that? Because with the default WM when 21.3 was installed, which was a compositing manager, using alt-tab to hop back and forth between two apps was unusably slow. When you hit alt-tab, a box with live screenshots of all open apps would appear and you could select the one you want to change to. Very pretty, but that box took way too long to pop up. My guess is, the computer would have to ask all the open apps to render themselves into a buffer somewhere in order to display that box, and it takes time. But then I saw a post that switching to a non-compositing window manager would fix it.

And it did -- I changed to Marco (without compositing) and alt-tab is nice and snappy again. It only shows icons of the apps instead of screenshots, but that's fine. BUT... now the screenshots of the apps that appear when I hover the mouse over the panel are usually wrong. Occasionally screenshots of the wrong app, more usually it looks like a photo of what the old-fashioned TV used to look like when your vertical hold (remember that?) went baa-ways. Or a blank black rectangle. Sometimes there's no screenshot, and I only get a line of text with the contents of the app window's title bar. And tbh that would be fine too, if that was what always happened; the screenshots are nice but not necessary. 

I have tried changing the window manager back, and also trying the other available WM options. It didn't fix the problem for me. I feel I'm missing something silly, but I don't know what.
